test/new-e2e/test-infra-definition/docker_test.go

```diff
@@ -9,18 +9,21 @@ import (
 	"fmt"
 	"regexp"
 	"testing"
+	"time"
 
 	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
 	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client"
-	"github.com/DataDog/test-infra-definitions/components/datadog/agent/dockerparams"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/params"
+	"github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/stretchr/testify/assert"
 )
 
 type dockerSuite struct {
 	e2e.Suite[e2e.DockerEnv]
 }
 
 func TestDocker(t *testing.T) {
-	e2e.Run(t, &dockerSuite{}, e2e.DockerStackDef(dockerparams.WithAgent()))
+	e2e.Run(t, &dockerSuite{}, e2e.DockerStackDef(os.AMD64Arch), params.WithDevMode())
```

Comment:
We should not keep dev mode here, right?

Reply:
Nice catch ! 

`WithDevMode` is disabled on the CI, but I didn't mean to keep this in code !